.github/workflows/bench.yml

```diff
@@ -34,10 +34,10 @@ jobs:
         run: cargo criterion --message-format json > criterion_output.log
 
       - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v1
+        uses: aws-actions/configure-aws-credentials@v4.0.1
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          role-to-assume: arn:aws:iam::024603541914:role/GitHubOIDCRole
+          role-session-name: s2ntlsghabenchsession
```

Comment:
Neither of these workflows are setup to run on PRs, so we won't actually know if this works until after it's merged. Not sure if it's worth temporarily modifying these to run on PRs so they can be tested? I guess worst case we can revert this if we see that they stop working?

Reply:
Correct, although because s2n-quic is using the same IdP, you can see a successful s3 publish in this workflow https://github.com/aws/s2n-quic/actions/runs/11336851180/job/31527471049?pr=2350 . Also correct: reverting will flip back to using the  older creds.